src/solver/conv_bin_winoRxS_fused.cpp

```diff
@@ -275,7 +269,16 @@ ConvSolution ConvBinWinogradRxSFused::GetSolution(const FusionContext& context,
     };
     return result;
 }
-
+float ConvBinWinogradRxSFused::GetWti(const FusionContext&, const FusionDescription& problem) const
+{
+    const auto conv_problem = problem.GetConvProblem(0, miopen::conv::Direction::Forward);
+    const auto x            = conv_problem.GetWeightsWidth();
+    const auto y            = conv_problem.GetWeightsHeight();
+    if(x == 3 && y == 3)
+        return 1.f;
+    else
+        return 5.f / 100.f;
```

Comment:
This function should probably be removed. The `ConvBinWinogradRxS` does not have it.

Reply:
Fusion is somewhat different and this is how they were selected before #2388. I would rather restore it to how it worked. Later it can be changed to any desired behavior. Otherwise performance would be different from what is expected (to better or worse).